### ISSUE TITLE
Setup default template for one/multiple images

### ISSUE THREAD

**Issue by noppelmax** [vspaceone/spaceinfo#27]

**Comment by H3wastooshort** [comment]
maybe allow for an option in the config.ini for ex.:
```ini
...
template="1picture.html"

[template_options]
pic_url="pic.png"
``` 
this uses the `1picture.html` file from the `_templates` directory located at the root of the pages directory

**Comment by noppelmax** [comment]
I thought about simply
```ini
pics= pic1.png pic2.png
```
But yes, having multiple options for template might be helpful in the future...

**Comment by H3wastooshort** [comment]
how are you planning to hand the configuration over? Just supplying a static html with javascript to read the config out of the .ini on its own would be the simplest but maybe not that efficient

**Comment by noppelmax** [comment]
?

**Comment by H3wastooshort** [comment]
how does the template know the options set the config.ini? is it included into the template by the server before the file is sent to the client or does it have to fetch() the config.ini itself and then parse it?

example for serverside:
`<!-- pic_url -->` -> `https://http.cat/200.jpg`

**Comment by noppelmax** [comment]
Yes sure, the server is populating the templates with values from the `config.ini`. I'm on it. :)

**Comment by H3wastooshort** [comment]
how would you do the syntax? funky HTML comments like `<!--{config_name_here}-->` that get replaced by the actual value?

i want to make a few templates and having the syntax defined would allow me to stop [guessing dumb solutions](https://pastebin.com/H2kvWzkf)

**Comment by noppelmax** [comment]
https://palletsprojects.com/p/jinja/

**Comment by noppelmax** [comment]
But you have to pass the wanted value explicitely in the template call e.g. `flask.render_template('home.html', shows = getShows(), version=VERSION)`

**Comment by noppelmax** [comment]
https://flask.palletsprojects.com/en/2.1.x/tutorial/templates/
